studio-docs/source/org/graphs.mdx

```diff
@@ -60,6 +66,12 @@ You can configure Apollo Server to associate the metrics it sends to Apollo Stud
 
 > Make sure you associate metrics with the correct variant! Otherwise, metrics from your staging and test environments will be included in reports for your production graph.
 
+### Protected variants (Enterprise only)
+
+If you have an Enterprise plan, you can designate particular variants of a graph as **protected variants**. Members of your organization with the [`Contributor` role](/org/members/#organization-wide-member-roles) _cannot_ push schema updates to a protected variant. Only members with the `Organization Admin` or `Graph Admin` role can do so. `Contributor` members still have read access to protected variants.
```

Comment:
Shall we make this complete? The other two things forbidden to Contributors on protected variants are:
- Ability to manage Explorer settings like the URL
- Ability to report usage (graph keys only)